python/flask/webappWithTests/Tests/functional_tests/test_functional.py

```diff
@@ -0,0 +1,30 @@
+import pytest
```

Comment:
same comments as the other pr, i believe the code is almost the same

Reply:
Fixed assertion  for selenium test. Added comment